server/src/metadata/field-metadata/field-metadata.entity.ts

```diff
@@ -7,36 +9,55 @@ import {
   PrimaryGeneratedColumn,
   UpdateDateColumn,
 } from 'typeorm';
+import { Authorize, IDField, PagingStrategies, QueryOptions, Relation } from '@ptc-org/nestjs-query-graphql';
 
 import { ObjectMetadata } from 'src/metadata/object-metadata/object-metadata.entity';
 
 export type FieldMetadataTargetColumnMap = {
   [key: string]: string;
 };
 
+@ObjectType('field')
+@QueryOptions({
+  defaultResultSize: 10,
+  maxResultsSize: 100,
+  disableFilter: true,
+  disableSort: true,
+})
+@Authorize({
```

Comment:
See https://tripss.github.io/nestjs-query/docs/graphql/authorization#authorize-decorator
Note: This act as a filter according to the doc so it will simply return results that match this but we can also throw an AuthorizationException if needed